packages/framer-motion/src/animation/__tests__/animate-waapi.test.ts

```diff
@@ -12,6 +13,8 @@ describe("animate() with WAAPI", () => {
             { duration: 1, transform: { duration: 2 } }
         )
 
+        await nextFrame()
```

Comment:
WAAPI `nextFrame` calls generally added as the WAAPI animation is only initialised after keyframes are resolved. We could add more custom logic here to say if something is WAAPI we can resolve fewer keyframes as it can handle CSS vars etc itself